common/src/main/java/io/seata/common/metadata/Metadata.java

```diff
@@ -0,0 +1,118 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.common.metadata;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import io.seata.common.store.StoreMode;
+import io.seata.common.util.StringUtils;
+
+
+import static io.seata.common.DefaultValues.DEFAULT_RAFT_PORT_INTERVAL;
+import static io.seata.common.DefaultValues.DEFAULT_SEATA_GROUP;
+
+/**
+ * @author funkye
+ */
+public class Metadata {
+
```

Comment:
MetaData need term info when start a new election term.The time is unreliable.

Reply:
> MetaData need term info when start a new election term.The time is unreliable.

Please transfer to view #5226